opennms-doc/guide-admin/src/asciidoc/text/ticketing/tsrm.adoc

```diff
@@ -56,9 +56,18 @@ Following table shows mapping between OpenNMS ticket and TSRM Incident
 | `summary`               | `DESCRIPTION`
 | `details`               | `DESCRIPTIONLONGDESCRIPTION`
 | `user`                  | `REPORTEDBY`
-| `siteId`                | `SITEID`
-| `location`              | `LOCATION`
+| `affectedPerson`        | `AFFECTEDPERSON`
```

Comment:
Can we update the documentation to clearly differentiate between actual ticket fields like summary (ticket.getSummary()) and those that are pulled from attributes i.e. ticket.getAttribute("location")?